background.ts

```diff
@@ -746,3 +746,20 @@ function newK8sManager() {
 
   return mgr;
 }
+
+/**
+ * Implement the methods the HttpCommandServer needs to service its requests.
+ * These methods should be thin wrappers around existing functionality in the rest of the backend.
+ * Getters normally return strings, either JSON strings or scalars.
+ * Setters normally return either `true | false`, or possibly `true | error`.
+ * The `requestShutdown` is a special case that never returns .
```

Comment:
github deleted my comment. Not going to write it all out, but if `Electron.app.quit()` can return, the code has to predict that will happen. Because if we wait to determine if it succeeded, it's too late to write back a 202 success code to the response object.